edge.go

```diff
@@ -190,9 +190,10 @@ func (e *Edge) CollectBatch(b models.Batch) error {
 // Increment the emitted count of the group for this edge.
 func (e *Edge) incEmitted(group models.GroupID, tags models.Tags, dims []string) {
 	e.groupMu.Lock()
-	defer e.groupMu.Unlock()
```

Comment:
This is great, do you want to add a comment as to why its not in a defer so that we don't change it back at a later date. Thanks